### DISCUSSION TITLE
Creating Vivado project

### DISCUSSION THREAD

**Discussion by kristopher391** [openhwgroup/cva6#1386] [Q&A]
Hello,
I am a student working on the CVA6. I am new to this processor and I would like to run a synthesis of it on Vivado. I already cloned the git but I don't know what to do next. Can anyone tell me the steps to follow to get the Vivado project working please ?

Thanks for your answers.

Kristopher

**Comment by aitesam961** [comment]
Hi @kristopher391, 
Once you have created the project, you need to load the `RTL` along with `packages` and `headers` into the vivado `Design Sources`. There are quite a few issues down the road that need to be solved.
